VimeoNetworking/Sources/Request+Cache.swift

```diff
@@ -32,12 +32,10 @@ public extension Request
     /// Generates a unique cache key for a request, taking into account endpoint and parameters
     var cacheKey: String
     {
-        var cacheKey = "cached" + self.path
+        let url = NSURL(string: self.path)
+        let urlPath = url?.path ?? ""
         
-        if let parameters = self.parameters
-        {
-            cacheKey = cacheKey + "." + String(parameters.description.hashValue)
-        }
+        var cacheKey = "cached" + urlPath + "." + String(self.path.hashValue)
```

Comment:
OK so I see we're ignoring the parameters. Is that because we're assuming anything we cache will be a `GET`, and therefore shouldn't theoretically have a request body? I think this makes sense, but maybe we should throw some guards with asserts up top to make sure that `self.method == GET` and `self.parameters == nil`.